examples/diffantom/sticks2dwis

```diff
@@ -233,15 +242,25 @@ lambda1 = args.lambda1
 lambda2 = args.lambda2
 model = GaussianModel(lambda1, lambda2)
 signal_operators = {}
-tau = 50e-3  # diffusion time: 50ms
-unique_bvals = set(bvals[indices_dwis].tolist())
 gradient_theta = np.arccos(gradient_directions[:, 2])
 gradient_phi = np.arctan2(gradient_directions[:, 1], gradient_directions[:, 0])
 H = shm.matrix(gradient_theta, gradient_phi, order=order_sh)
-for bval in unique_bvals:
-    rs = model.signal_convolution_sh(order_sh, bval)
-    indices = (gradient_bvals == bval)
-    H[indices] *= rs[np.newaxis, :]
+if b_perps == None :
+    unique_bvals = set(bvals[indices_dwis].tolist())
+    for bval in unique_bvals :
+        rs = model.signal_convolution_sh(order_sh, bval)
+        indices = (gradient_bvals == bval)
+        H[indices] *= rs[np.newaxis, :]
+else :
+    bvals_reshaped = bvals[indices_dwis].reshape((bvals[indices_dwis].shape[0], 1))
+    bperps_reshaped = bperps[indices_dwis].reshape((bperps[indices_dwis].shape[0], 1))
+    bvals_bperps = np.append(bvals_reshaped, bperps_reshaped, axis = 1)
+    bvals_bperps = [tuple(element) for element in bvals_bperps]
```

Comment:
I think these lines (255-258) can be replaced using the builtin function `zip` of python (something like `bvals_bperps = zip(bvals[indices_dwis], bperps[indices_dwis])`)